python/pyarrow/_dataset.pyx

```diff
@@ -92,6 +92,22 @@ cdef class Dataset:
         else:
             return Expression.wrap(expr)
 
+    def get_fragments(self, columns=None, filter=None):
+        """Returns an iterator over the fragments in this dataset.
+
+        Parameters
+        ----------
+        columns : list of str, default None
+            List of columns to project.
```

Comment:
The columns keyword can probably be removed here? (since it can never have effect?)

Reply:
Ah, didn't see your last comment regarding this